Python/Product/Common/Infrastructure/CancellationTokens.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.PythonTools.Infrastructure {
     public static class CancellationTokens {
```

Comment:
I thought the same thing initially :) Each call to the getter will get you a new token (they're not singletons), so it's fine.

---
In reply to: [108528637](https://github.com/Microsoft/PTVS/pull/2323#discussion_r108528637) [](ancestors = 108528637)